docs/section5/Rule5-53.md

```diff
@@ -19,40 +17,40 @@
+        - Check if subsurface is opaque door: ```if ( subsurface_b.classification == "DOOR" ) AND ( subsurface_b.glazed_area <= subsurface.opaque_area )```
 
-        - Else if surface is exterior mixed, get baseline construction for both residential and non-residential type door: `else if ( scc_b == "EXTERIOR MIXED" ): target_u_factor_res = data_lookup(table_G3_4, climate_zone, "EXTERIOR RESIDENTIAL", "DOOR", subclassification_b), target_u_factor_nonres = data_lookup(table_G3_4, climate_zone, "EXTERIOR NON-RESIDENTIAL", "DOOR", subclassification_b)`
+          - Get subsurface subclassification: ```subclassification_b = subsurface_b.subclassification```
 
-          - If residential and non-residential type door construction requirements are the same, save as baseline construction: `if target_u_factor_res == target_u_factor_nonres: target_u_factor = target_u_factor_res`
+          - If surface is exterior residential, exterior non-residential, or semi-exterior, get baseline construction from Table G3.4-1 to G3.4-8 based on climate zone, surface conditioning category and door type: `if ( ( scc_b == "EXTERIOR RESIDENTIAL" ) OR ( scc_b == "EXTERIOR NON-RESIDENTIAL" ) OR ( scc_b == "SEMI-EXTERIOR" ) ): target_u_factor = data_lookup(table_G3_4, climate_zone, scc_b, "DOOR", subclassification_b)`
 
-          - Else: `manual_review_flag = TRUE`
+          - Else if surface is exterior mixed, get baseline construction for both residential and non-residential type door: ```else if ( scc_b == "EXTERIOR MIXED" ): target_u_factor_res = data_lookup(table_G3_4, climate_zone, "EXTERIOR RESIDENTIAL", "DOOR", subclassification_b), target_u_factor_nonres = data_lookup(table_G3_4, climate_zone, "EXTERIOR NON-RESIDENTIAL", "DOOR", subclassification_b)```
 
-          **Rule Assertion:**  
+            - If residential and non-residential type door construction requirements are the same, save as baseline construction: ```if target_u_factor_res == target_u_factor_nonres: target_u_factor = target_u_factor_res```
 
-          - Case 1: For each opaque door, if the door's parent zone has both residential and non-residential spaces and the construction requirements for door are different, request manual review: `if manual_review_flag == TRUE: UNDETERMINED and raise_message "ZONE HAS BOTH RESIDENTIAL AND NON-RESIDENTIAL TYPE SPACES AND THE REQUIREMENT FOR U-FACTOR FOR DOORS ARE DIFFERENT. VERIFY DOOR U-FACTOR IS MODELED CORRECTLY."`
+            **Rule Assertion:**  
+            - Case 1: If the door's subclassification is not SWINGING_DOOR or NONSWINGING_DOOR, outcome is UNDETERMINED: ```if subclassification_b not in ["SWINGING_DOOR","NONSWINGING_DOOR"]: outcome = UNDETERMINED```
```

Comment:
I suggest that we provide FAIL outcome if door U-factor doesn't match prescribed values for either door types, similar to what we do in Case 3 for doors with unclear residential vs non-residential conditioning category status. Also, in some cases U-factor requirements are the same for both types of doors (e.g., residential in CZ6)- for these cases, we are able to provide a definitinve pass/fail outcome. Also, for the Fail outcome, please provide a message indicating whether the modeled U-factor is lower than prescribed (a conservative FAIL) or higher. @supriyagoel , please share the latest on how to handle this. (I think the plan is to drop the AHJ Conservative flag, right?)

Reply:
@mkarpman and @JacksonJ-KC in the case the U factor is lower than the prescribed value, please simply include a text in the outcome message that states, 'rule evaluation fails with a conservative outcome'. The RCT will be able to filter based on this outcome message and provide a count  of rules evaluations that include this message